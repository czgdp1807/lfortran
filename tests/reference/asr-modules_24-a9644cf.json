{
    "basename": "asr-modules_24-a9644cf",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/modules_24.f90",
    "infile_hash": "0fb3167aa8f4f57f5ddb70eaf807e1b239ff4edce2562d8cb0e69b29",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-modules_24-a9644cf.stderr",
    "stderr_hash": "3014697c3e5f266f0dfdc09f40267b057c87db704511c39ce8910156",
    "returncode": 2
}