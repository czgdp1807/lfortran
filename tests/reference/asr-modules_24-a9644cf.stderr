semantic error: Derived type `null` is not defined
  --> tests/../integration_tests/modules_24.f90:11:48
   |
11 |        type(toml_table), pointer :: current => null()
   |                                                ^^^^^^ 
